xml/System.Web/SameSiteMode.xml

```diff
@@ -54,7 +67,7 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>No mode is specified.</summary>
+        <summary>Emits "SameSite=None" (see remarks).</summary>
```

Comment:
This summary doesn't align with the others. They describe the resulting browser behavior, where this describes the cookie text.
```suggestion
        <summary>The cookie will be sent with all requests (see remarks).</summary>
```